config/index.ts

```diff
@@ -1,19 +1,19 @@
 const config = {
-  site_title: "AlexCode",
-  site_description:
-    "Technologies suck, they won't be if you read my blog 💪\nOn my blog I'm sharing tips to increase your productivity and developer happiness",
-  site_keywords: ["Programming post", "Blog about programming"],
-  author_name: "Alex",
-  author_position: "Full Stack JavaScript Developer",
-  social_links: [
-    { link: "https://t.me/HeWorksSoHard", image: "/images/telegram.svg", width:20.99,height:16.99},
-    { link: "https://github.com/1alexvash", image: "/images/github-gray.svg",width:19,height:19 },
-    { link: "mailto:1alexvash@gmail.com", image: "/images/mail.svg",width:24,height:15.99 },
-  ],
-  host_url: "https://alex-code.com/",
-  defaultImage:
-    "https://alex-code.com/post-images/2022-08-05-book-review-so-good-they-cant-ignore-you.webp",
-  posts_per_page: 12,
+	site_title: "AlexCode",
+	site_description:
+		"Technologies suck, they won't be if you read my blog 💪\nOn my blog I'm sharing tips to increase your productivity and developer happiness",
+	site_keywords: ["Programming post", "Blog about programming"],
+	author_name: "Alex",
+	author_position: "Full Stack JavaScript Developer",
+	social_links: [
+		{ link: "https://t.me/HeWorksSoHard", image: "/images/telegram.svg", width: 20.99, height: 16.99 },
+		{ link: "https://github.com/1alexvash", image: "/images/github-gray.svg", width: 19, height: 19 },
+		{ link: "mailto:1alexvash@gmail.com", image: "/images/mail.svg", width: 24, height: 15.99 },
+	],
+	host_url: "https://alex-code.com/",
+	defaultImage:
+		"https://alex-code.com/post-images/2022-08-05-book-review-so-good-they-cant-ignore-you.webp",
+	posts_per_page: 12,
```

Comment:
@ntorbinskiy file is still not formatted
![animation](https://user-images.githubusercontent.com/46556479/193411207-4bb57dbd-1404-4bb8-aa14-f91ba73d987c.gif)

It gets annoying, to review the same thing over and over again for 5 times

Reply:
I'm sorry, there was  accidently turned on another formatter.  I've fixed it in the next commit.

Reply:
@ntorbinskiy maybe this other formatter is causing problems with break lines on the SEO branch?